src/js/themes/base.js

```diff
@@ -1041,32 +1041,38 @@ export const generate = (baseSpacing = 24, scale = 6) => {
       // extend: undefined,
     },
     notification: {
+      // any box props
       container: {
         pad: { horizontal: 'small', vertical: 'xsmall' },
         background: {
           color: 'background-front',
         },
       },
       toast: {
+        // any box props
         container: {
           elevation: 'medium',
           round: 'xsmall',
           width: 'medium',
         },
       },
+      // any box props
       iconContainer: {
         pad: { right: 'small' },
```

Comment:
In this context (documenting properties of an object) the comments should be inside where the properties would reside. Applies to all.

```suggestion
      iconContainer: {
        // any box props
        pad: { right: 'small' },
```